docs/how-to/how-to-set-sandbox-config-kata.md

```diff
@@ -94,6 +94,8 @@ There are several kinds of Kata configurations and they are listed below.
 | `io.katacontainers.config.hypervisor.virtio_fs_extra_args` | string | extra options passed to `virtiofs` daemon |
 | `io.katacontainers.config.hypervisor.enable_guest_swap` | `boolean` | enable swap in the guest |
 | `io.katacontainers.config.hypervisor.use_legacy_serial` | `boolean` | uses legacy serial device for guest's console (QEMU) |
+| `io.katacontainers.config.hypervisor.default_gpus` | uint32 | the minimum number of GPUs required for the VM. Only used by remote hypervisor to help with instance selection |
+| `io.katacontainers.config.hypervisor.default_gpu_model` | string | the GPU model required for the VM. Only used by remote hypervisor to help with instance selection |
```

Comment:
@bpradipt Is the GPU model name depending on the cloud provider?
Could we accept an array of names that would be tried in order, so that you could have a configuration that works for multiple clouds?

Reply:
We can have more flexibility in the cloud-api-adaptor implementation. Also for additional models may be having another specific annotation will be helpful. But we can handle it in future based on requirements